packages/front-end/components/Settings/UpgradeModal/index.tsx

```diff
@@ -257,6 +285,115 @@ export default function UpgradeModal({ close, source }: Props) {
     }
   };
 
+  const upgradeHeader = (
+    <>
+      <h3 className="mb-1">Upgrade to Pro</h3>
+      <p
+        className="mb-0"
+        style={{ color: "var(--color-text-mid", fontSize: "16px" }}
+      >
+        Get instant access to advanced experimentation, permissioning and
+        security features.
+      </p>
+    </>
+  );
+
+  function trialAndUpgradeTreatment() {
+    return (
+      <div>
+        {upgradeHeader}
+        <div className="py-4">
+          <RadioCards
+            columns="2"
+            value={trialAndUpgradePreference}
+            setValue={(v) => setTrialAndUpgradePreference(v)}
+            options={[
+              {
+                value: "trial",
+                label: "Pro Trial",
+                description: "Free for 14 days",
+              },
+              {
+                value: "upgrade",
+                label: "Pro",
+                description: "Upgrade now. No credit card required.",
+              },
+            ]}
+          />
+        </div>
+        <p style={{ color: "var(--color-text-mid" }}>
+          Continuing will{" "}
+          {trialAndUpgradePreference === "trial"
+            ? "start a 14-day Pro plan trial for your organization."
+            : "require a credit card to purchase a Pro plan for your organization."}
+          <a
+            href="https://www.growthbook.io/pricing"
+            className="text-decoration-none pl-1"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <strong className="a link-purple text-decoration-none">
+              Learn more <FaExternalLinkAlt className="pl-1" />
+            </strong>
+          </a>
+        </p>
+      </div>
+    );
+  }
+
+  function upgradeOnlyTreatment() {
+    return (
+      <div>
+        {upgradeHeader}
+        <div className="py-4">
+          <Flex align="center" className="pb-2">
+            <FaCheckCircle className="mr-2" color="blue" />
+            <strong>Up to 100 team members</strong>
+          </Flex>
+          <Flex align="center" className="pb-2">
+            <FaCheckCircle className="mr-2" color="blue" />
+            <strong>Encrypted SDK endpoint response</strong>
+          </Flex>
+          <Flex align="center" className="pb-2">
+            <FaCheckCircle className="mr-2" color="blue" />
+            <strong>
+              Advanced experimentation: CUPED, Sequential testing, Bandits and
+              more
+            </strong>
+          </Flex>
```

Comment:
I'm going to play with these bullet items and see if I can come up with better ones to highlight for Pro.  These (especially the first 2) are not very compelling reasons to upgrade.